lstchain/datachecks/containers.py

```diff
@@ -283,6 +310,42 @@ def fill_pixel_wise_info(self, table, mask, histogram_binnings,
                      bins=histogram_binnings.hist_pixelchargespectrum)
         self.hist_pixelchargespectrum = counts
 
+        # Find bright stars (mag<=8 within 3 deg of telescope pointing) and
+        # count how many of them are close to each pixel:
+
+        # Just use the time in the middle of the subrun, from the sampled times:
+        sampled_times = self.dragon_time
+        obstime = Time(sampled_times[int(len(sampled_times)/2)],
+                       scale='utc', format='unix')
+        horizon_frame = AltAz(location=location, obstime=obstime)
+        pointing = SkyCoord(az=self.mean_az_tel*u.rad,
```

Comment:
See? just store the coordinate as astropy coordinate above and you don't have to reconstruct the same object here.